client.go

```diff
@@ -0,0 +1,44 @@
+package llblib
+
+import (
+	"context"
+	"net"
+
+	dockerclient "github.com/docker/docker/client"
+	"github.com/moby/buildkit/client"
+	_ "github.com/moby/buildkit/client/connhelper/dockercontainer"
+	_ "github.com/moby/buildkit/client/connhelper/kubepod"
+	"github.com/pkg/errors"
+)
+
+func NewClient(ctx context.Context, addr string, opts ...client.ClientOpt) (*client.Client, error) {
+	cln, err := newClient(ctx, addr, opts)
+	if err != nil {
+		return nil, err
+	}
+	// if _, err = cln.ListWorkers(ctx); err != nil {
```

Comment:
I added in the call to `Info` (which I assume is what you meant).  It is a tad awkward because buildkit in docker does not implement that, I assume it is too old.  So I look for the grpc error for missing Info api, then call back to ListWorkers.

Reply:
Yes, `Info`, sorry.

Didn't realize it was a new RPC.

Reply:
Yeah, not sure how new it is, maybe they just dont expose it in docker buildkit for some reason 🤷